tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsContextPropagator.cs

```diff
@@ -31,7 +36,30 @@ public void Inject<TCarrier>(PathwayContext context, TCarrier headers)
     {
         if (headers is null) { ThrowHelper.ThrowArgumentNullException(nameof(headers)); }
 
-        headers.Add(DataStreamsPropagationHeaders.PropagationKey, PathwayContextEncoder.Encode(context));
+        var encodedBytes = PathwayContextEncoder.Encode(context);
+
+        int base64Length = ((encodedBytes.Length + 2) / 3) * 4;
```

Comment:
Could you add a comment here for why we are doing `+2 / 3 * 4`? As a reader I can't easily tell

 `((encodedBytes.Length + 2) / 3) * 4;`